pipeline/beam_tables.py

```diff
@@ -59,26 +62,46 @@
 EMPTY_GZIPPED_FILE_SIZE = 33
 
 
-def _get_existing_datasources(table_name: str, project: str) -> List[str]:
-  """Given a table return all sources that contributed to the table.
+def _get_existing_datasources(destination: str, project: str) -> List[str]:
+  """Given a BigQuery or GCS destination return all contributing sources.
 
   Args:
-    table_name: name of a bigquery table like
+    destination: name of a bigquery table like
```

Comment:
I would not overload this field to have two different types of strings. Instead I would break this out into two methods  `_get_existing_bq_datasources` and `_get_existing_file_datasources`

Reply:
done